client/src/components/kvEditor/KVEditorRow.tsx

```diff
@@ -195,6 +202,16 @@ function KVEditorRow({
     <div key={`${name}-${i}`} className={styles.row}>
       {!readOnly ? (
         <>
+          {canDisableRow && (
+            <input
+              type="checkbox"
+              className={cn(styles, 'checkbox', [colorMode])}
+              checked={isEnabled}
```

Comment:
The checkbox should either not be visible or at least disabled for the last row (empty row - it cannot be disabled)